<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyNetworkBlocker</string>
  </property>
  <widget class="QPushButton" name="ifaceButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Search Interfaces</string>
   </property>
  </widget>
  <widget class="QListWidget" name="devicesList">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>256</width>
     <height>251</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="devicesButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Ağdaki Cihazları Ara</string>
   </property>
  </widget>
  <widget class="QPushButton" name="arpButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Engelle</string>
   </property>
  </widget>
  <widget class="QListWidget" name="ifaceList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="stop">
   <property name="geometry">
    <rect>
     <x>436</x>
     <y>300</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Durdur</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ifaceButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
